change/office-ui-fabric-react-2020-03-06-14-36-14-master.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "comment": "Fixing drag/drop issues with Groups in DetailsList",
```

Comment:
Please write a more descriptive comment and preface with "DetailsList: " as these go out in release notes to our customers